rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/RMController.java

```diff
@@ -170,7 +193,11 @@ public void onSuccess(String result) {
                         loadingMessage.destroy();
                     }
                     Settings.get().clearSetting(SESSION_SETTING);
-                    RMController.this.loginPage = new LoginPage(RMController.this, null);
+                    if (domains == null || domains.isEmpty()) {
+                        getDomains(null);
```

Comment:
Because the getDomains method opens a LoginPage that needs to receive a message in the constructor:
`new LoginPage(RMController.this, message, domains);`